k8s/infrastructure/grafana-deployment.yaml

```
@@ -0,0 +1,41 @@
+            - name: GF_LOKI_URL
+              value: http://crm-loki:3100
+      volumes:
+        - name: grafana-dashboard
+          persistentVolumeClaim:
+            claimName: grafana-dashboard-pvc
+        - name: grafana-provisioning-dashboards
+          configMap:
+            name: grafana-provisioning-dashboards-config
+        - name: grafana-provisioning-datasources
+          configMap:
+              name: grafana-provisioning-datasources-config
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 12 but found 14

(indentation)